src/draw.c

```diff
@@ -138,28 +142,54 @@ static SDL_Rect createPlayer(void)
         assert(0);
         exit(-1);
     }
-
-    return playerEntity;
 }
 
 #define PLAYER_SHIP_SPEED 10
 
 int MovePlayerLeft(void)
```

Comment:
All 4 could be one general function with switch case, but I dont know whether that's the best approach